README.md

```diff
@@ -19,3 +19,29 @@ of the production JupyterHubs.
 ## Images
 
 User images for each of the hubs are also managed in this repository.
+
+## Procedure
+
+To make changes, fork this repo and [create a pull
+request](https://help.github.com/articles/about-pull-requests/). The
+choice for `base` in the GitHub PR user interface should be the staging
+branch of this repo while the choice for `head` is your fork. The pull
+request will trigger a [https://travis-ci.org/](Travis CI) process and
+potentially a rebuild of docker images depending on what modifications
+have been proposed. You can observe the status of the CI process by
+visiting its link on the page for your PR.
+
+Once this is complete and if there are no problems, you can request that
+someone review the PR before merging, or you can merge yourself if you
+are confident. This merge will trigger another Travis step which
```

Comment:
You can see the travis builds on https://travis-ci.org/berkeley-dsep-infra/datahub/builds